include/mbedtls/mbedtls_config.h

```diff
@@ -129,7 +129,11 @@
  * MBEDTLS_PLATFORM_TIME_MACRO, MBEDTLS_PLATFORM_TIME_TYPE_MACRO and
  * MBEDTLS_PLATFORM_STD_TIME.
  *
- * Comment if your system does not support time functions
+ * Comment if your system does not support time functions.
+ *
+ * \note If MBEDTLS_TIMING_C is set - to enable the semi-portable timing
+ *       interface - timing.c will include time.h regardless of the setting
+ *       of MBEDTLS_HAVE_TIME, unless MBEDTLS_TIMING_ALT is used.
```

Comment:
This note is not really important and not even completely correct: `timing.c` doesn't include `time.h` on Windows.

Reply:
I clarified that this is correct for `suitable` platforms, as I didn't want to go into more detail.